src/components/LoanFinding/LendingCategorySection.vue

```diff
@@ -125,10 +133,39 @@ export default {
 			}
 			return '336px';
 		},
+		multipleAmount() {
+			let amount = 0;
+			for (let index = 0; index < this.totalLoans; index += 1) {
+				const loan = this.loans[index];
+				const { unreservedAmount } = loan;
+
+				if (this.enableFiveDollarsNotes) {
+					amount += this.userBalance > 20 ? 25 : 5;
+				} else {
+					amount += unreservedAmount > 25 ? 25 : Number(unreservedAmount);
+				}
+			}
+			return amount;
+		},
+		totalLoans() {
+			return this.loans.length;
+		}
 	},
 	methods: {
 		addToBasket(payload) {
 			this.$emit('add-to-basket', payload);
+		},
+		addMultipleLoans() {
+			for (let index = 0; index < this.totalLoans; index += 1) {
+				const { unreservedAmount } = this.loans[index];
+				const key = `relending-${index}`;
```

Comment:
Extract this string template into a method that can be used here and in the template.